src/wallet/wallet_rpc_server_commands_defs.h

```diff
@@ -182,24 +182,30 @@ namespace wallet_rpc
   {
     struct request_t
     {
-      uint32_t account_index;
+      uint32_t    account_index;
+      uint32_t    count;
       std::string label;
 
       BEGIN_KV_SERIALIZE_MAP()
         KV_SERIALIZE(account_index)
+        KV_SERIALIZE(count)
```

Comment:
KV_SERIALIZE_OPT(count, 1);
Otherwise clients not setting it (ie, all current clients) will cause the server to read some random amount.